app/tasks/supplier_tasks.py

```diff
@@ -0,0 +1,13 @@
+from app.api.services import (
+    key_values_service,
+    suppliers
+)
+from . import celery
+
+
+@celery.task
+def update_supplier_metrics():
+    supplier_metrics = suppliers.get_supplier_metrics()
```

Comment:
Is it better to remove supplier in the function name? Eg. `get_metrics()` ?

Reply:
changed